compiler/src/iree/compiler/Codegen/Common/DecomposeBatchMmt4DOps.cpp

```diff
@@ -130,6 +132,35 @@ void DecomposeBatchMmt4DOpsPass::runOnOperation() {
   MLIRContext *ctx = &getContext();
   auto funcOp = getOperation();
 
+  funcOp.walk([&](linalg::BatchMmt4DOp op) {
+    auto out = op.getDpsInitOperand(0)->get();
+    auto outType = out.getType().cast<RankedTensorType>();
+    // Tile only non unit batch dimensions with tile size equals to 1.
+    if (outType.getShape()[0] <= 1) {
+      return;
+    }
+    SmallVector<int64_t> tileSizes{1};
```

Comment:
Please use assignment syntax. The brace initialization with constructor syntax is dangerous. It could potentially lead to a bug, e.g.,

```
// Could invoke an intializer list constructor, or a two-argument constructor.
Frobber frobber{size, &bazzer_to_duplicate};

// Makes a vector of two doubles.
std::vector<double> fifty_pies{50, 3.14};
```

For more context, see https://abseil.io/tips/88

Reply:
Thanks:)